sql/updates/world/master/9999_99_99_17997_world.sql

```diff
@@ -0,0 +1,77 @@
+DROP TABLE IF EXISTS `areatrigger_template`;
+CREATE TABLE `areatrigger_template` (
+  `Id` int(10) unsigned NOT NULL,
+  `Type` tinyint(3) unsigned NOT NULL DEFAULT '0',
+  `Flags` int(10) unsigned NOT NULL DEFAULT '0',
+  `Data0` float NOT NULL DEFAULT '0',
+  `Data1` float NOT NULL DEFAULT '0',
+  `Data2` float NOT NULL DEFAULT '0',
+  `Data3` float NOT NULL DEFAULT '0',
+  `Data4` float NOT NULL DEFAULT '0',
+  `Data5` float NOT NULL DEFAULT '0',
+  `ScriptName` char(64) NOT NULL DEFAULT '',
+  `VerifiedBuild` int(10) unsigned NOT NULL DEFAULT '0',
+  PRIMARY KEY (`id`)
+) ENGINE=MyISAM DEFAULT CHARSET=utf8;
+
+DROP TABLE IF EXISTS `areatrigger_template_polygon_vertices`;
+CREATE TABLE `areatrigger_template_polygon_vertices` (
+  `AreaTriggerId` int(10) unsigned NOT NULL,
+  `Idx` int(10) unsigned NOT NULL,
+  `VerticeX` float NOT NULL DEFAULT '0',
+  `VerticeY` float NOT NULL DEFAULT '0',
+  `VerticeTargetX` float NOT NULL DEFAULT '0',
+  `VerticeTargetY` float NOT NULL DEFAULT '0',
+  `VerifiedBuild` int(10) unsigned NOT NULL DEFAULT '0',
+  PRIMARY KEY (`areatriggerId`,`Idx`)
+) ENGINE=MyISAM DEFAULT CHARSET=utf8;
+
+DROP TABLE IF EXISTS `areatrigger_template_splines`;
+CREATE TABLE `areatrigger_template_splines` (
+  `MoveCurveId` int(10) unsigned NOT NULL,
+  `Idx` int(10) unsigned NOT NULL,
+  `SplineX` float NOT NULL DEFAULT '0',
+  `SplineY` float NOT NULL DEFAULT '0',
+  `SplineZ` float NOT NULL DEFAULT '0',
+  `VerifiedBuild` int(10) unsigned NOT NULL DEFAULT '0',
+  PRIMARY KEY (`MoveCurveId`,`Idx`)
+) ENGINE=MyISAM DEFAULT CHARSET=utf8;
+
+DROP TABLE IF EXISTS `areatrigger_template_auras`;
+CREATE TABLE `areatrigger_template_auras` (
+  `AreaTriggerId` int(10) unsigned NOT NULL,
```

Comment:
Don't think so, i feel like spellmisc mostly handle AT movements & scaling, while AreaTriggerId handle AT shape & casted spell.

But i may be wrong, if we found any AreaTrigger where spell casted depend of spellmisc, i'll change it